src/components/Hero/Hero.tsx

```diff
@@ -160,13 +160,13 @@ export const Hero = chakra(
           ? { bgColor: backdropBackgroundColor }
           : { backgroundColor };
       } else if (heroType === "tertiary" || heroType === "fiftyFifty") {
-        backgroundImageStyle = { backgroundColor };
+        backgroundImageStyle = backgroundColor ? { bg: backgroundColor } : {};
```

Comment:
`backgroundColor` is a css prop, but was the issue that `bg` has more precedence? Or perhaps a mix of this and that the `contentBoxStyling` came before the `styles.content` object?

Reply:
It seems to be somewhat unknown, `bg` for content is overwritten by the `backgroundColor` added with `contentBoxStyling`, but `backgroundColor` does not overwrite `bg` in the parent component. By setting `backgroundImageStyle` to use `bg` instead, the spread operator overwrites the default value consistently.